src/app/framework-modules/elements/ct-textarea/component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input, Output, EventEmitter} from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { Field } from '../../formdata/form/form';
+import { FormData } from '../../formdata/form/formData';
+
+@Component({
+	selector: 'ct-textarea',
+	templateUrl: './component.html',
+	styleUrls: []
+})
+
+/*
+* Wrapper class for angular material.
+* Unpacks values from the model to render a textarea
+* @ouput - valueChange - value change emitter.
+*/
+export class CTTextareaComponent {
+	@Input() public field: Field;
+	@Input() public fd: FormData;
+	@Output() public valueChange = new EventEmitter<any>();
+
+	control:FormControl;
+	
+	ngOnInit() {
+		this.control = this.fd.formGroup.get(this.field.name) as FormControl;
+	}
+	valueChangeDetector($event){

```

Comment:
Another linting nitpick.  Above it's `blah() {` but here it's `blah(){`. Inconsistent.

Please turn on linter and make it part of your PR checks, so the PR checks would fail if the new code isn't linted.  That way we won't have to leave linting comments on PRs.